File: Data Structures & Algorithms/two-integer-sum/submission-1.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        for i, n in enumerate(nums):
            for j, s in enumerate(nums[i+1:]):
                if n + s == target:
                    return [i,j+i+1]
        
    

File: Data Structures & Algorithms/two-integer-sum-ii/submission-1.go
func twoSum(numbers []int, target int) []int {
	expecting := make(map[int]int)
	
	for i, num := range numbers {
		if index1, ok := expecting[num]; ok {
			return []int{index1, i+1}
		} else {
			expecting[target-num] = i+1
		}
	}
	return []int{}
}
